apis/management.cattle.io/v3/multi_cluster_app.go

```diff
@@ -39,3 +40,8 @@ type Answer struct {
 	ClusterName string            `json:"clusterName,omitempty" norman:"type=reference[cluster]"`
 	Values      map[string]string `json:"values,omitempty" norman:"required"`
 }
+
+type Member struct {
+	UserName   string `json:"userName,omitempty" norman:"type=reference[user],required"`
+	AccessType string `json:"accessType,omitempty" norman:"type=enum,options=everything|readOnly|upgrade"`
```

Comment:
lets use `all` instead of `everything`

Reply:
And `update` instead of `upgrade` as it should cover update and rollback actions